src/SFML/Main/MainAndroid.cpp

```diff
@@ -568,8 +568,7 @@ JNIEXPORT void ANativeActivity_onCreate(ANativeActivity* activity, void* savedSt
     sf::err().rdbuf(&states->logcat);
 
     // Launch the main thread
-    sf::Thread* thread = new sf::Thread(sf::priv::main, states);
-    thread->launch();
+    std::thread(sf::priv::main, states).detach();
```

Comment:
Did we before really just leak the thread?
Should this really be just detached? 🤔

Reply:
> Did we before really just leak the thread?
Yes.

> Should this really be just detached?
Probably not. Outside of the scope of this PR, but definitely something to look into.